src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -323,7 +323,11 @@ export default class OverflowMenu extends Component {
   };
 
   closeMenu = () => {
+    let wasOpen = this.state.open;
     this.setState({ open: false }, () => {
+      if (wasOpen) {
+        this.focusMenuEl();
```

Comment:
Did you want to remove: https://github.com/Kyle-Cooley/carbon-components-react/blob/742a24bb891752975242ccc388f49f22515c0948/src/components/OverflowMenu/OverflowMenu.js#L383-L386